README.md

```diff
@@ -1,3 +1,15 @@
+![Leanplum](Leanplum.png)
+
+<p align="center">
+<img src='https://jenkins.leanplum.com/buildStatus/icon?job=apple-sdk-github'></a>
+<img src="https://img.shields.io/cocoapods/dt/Leanplum-iOS-SDK.svg?maxAge=3600" alt="Downloads" />
+<img src="https://img.shields.io/badge/platform-iOS-blue.svg?style=flat" alt="Platform iOS" />
+<img src="https://img.shields.io/badge/platform-tvOS-blue.svg?style=flat" alt="Platform tvOS" />
```

Comment:
Yeah i guess. Both PRs are there so we will see which one is ready to be merged first :)